server/sonar-db-dao/src/main/java/org/sonar/db/qualityprofile/ActiveRuleDao.java

```diff
@@ -21,9 +21,11 @@
 
```

Comment:
The commit comment was misleading to me. I would suggest: `profile related SQLs to delete rows by list of keys`